internal/cli/default_setter_opts.go

```diff
@@ -157,7 +157,7 @@ func (opts *DefaultSetterOpts) orgs(filter string) (results interface{}, err err
 		if r.GetTotalCount() > resultsLimit {
 			return nil, errTooManyResults
 		}
-		results = r.Results
+		results = *r.Results
```

Comment:
current function returns (results interface{}, err error) and the compiler was not catching that we were returning either an array or an array pointer.
With the mongocli fork, this issue will be properly solved (as we'll return a well-know type instead of `any`)